### ISSUE TITLE
Doesn't compile with latest master

### ISSUE THREAD

**Issue by theoparis** [Snektron/vulkan-zig#55]
```
/mnt/data/projects/vulkan-zig/examples/triangle.zig:241:9: error: pointless discard of function parameter
    _ = extent;
        ^~~~~~
/mnt/data/projects/vulkan-zig/examples/triangle.zig:284:23: note: used here
            .extent = extent,
                      ^~~~~~
/mnt/data/projects/vulkan-zig/examples/triangle.zig:242:9: error: pointless discard of function parameter
    _ = buffer;
        ^~~~~~
/mnt/data/projects/vulkan-zig/examples/triangle.zig:297:81: note: used here
        gc.vkd.cmdBindVertexBuffers(cmdbuf, 0, 1, @ptrCast([*]const vk.Buffer, &buffer), &offset);
                                                                                ^~~~~~
/mnt/data/projects/vulkan-zig/examples/triangle.zig:243:9: error: pointless discard of function parameter
    _ = pool;
        ^~~~
/mnt/data/projects/vulkan-zig/examples/triangle.zig:251:48: note: used here
    errdefer gc.vkd.freeCommandBuffers(gc.dev, pool, @truncate(u32, cmdbufs.len), cmdbufs.ptr);
                                               ^~~~
/mnt/data/projects/vulkan-zig/examples/triangle.zig:244:9: error: pointless discard of function parameter
    _ = gc;
        ^~
/mnt/data/projects/vulkan-zig/examples/triangle.zig:301:13: note: used here
        try gc.vkd.endCommandBuffer(cmdbuf);
            ^~

```

**Comment by theoparis** [comment]
Seems like its a easy fix - just removing the discard lines seems to fix it.

**Comment by Snektron** [comment]
Fixed in 80a201f
